src/ignitus-Routes/Styles/style.scss

```diff
@@ -1,19 +1,38 @@
 // import ignitus token variables
 @import './src/shared/styles/ignitus-tokens.scss';
 
+
 .container-404 {
     min-height: 60vh;
     width: 100vw;
```

Comment:
Change `width` to `100%` because the `100vw` adds a bit more than the width of the screen. That's why it's having horizontal scrollbar at the bottom.

Reply:
@gittysachin correct I somehow missed it :)